### DISCUSSION TITLE
Add option to clear the internal cache in the Directus app

### DISCUSSION THREAD

**Discussion by abdonrd** [directus/directus#17193] [Feature Requests]
Directus has an option to clear the internal cache:
https://docs.directus.io/reference/system/utilities.html#clear-the-internal-cache

I think it would be great if there was an option from the Directus app to be able to run it. Obviously only for users who are administrators.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
